packages/reket-core/tests/config/client.spec.js

```diff
@@ -0,0 +1,29 @@
+import { ReketConfigClient } from '../../src/config/client';
+import { ReketConfigItem } from '../../src/config/item';
+
+import { MockReketClient } from '../mocks';
+
+describe('ReketConfigClient implementation', () => {
+  test('it should instanciate a ReketConfigClient instance', () => {
+    const clientConfig = new ReketConfigClient();
+
+    // check if clientConfig is well inherited
+    expect(clientConfig instanceof ReketConfigItem).toBe(true);
+    expect(clientConfig.value).toBe(undefined);
+  });
```

Comment:
I think this test doesn't is not relevant and should be removed. 

We are testing that the "new" keyword is working correctly and that the "extends" works correctly too. That's just javascript, we shouldn't test that Javascript works.